Airbnb.Domain/Entities/Booking.cs

```diff
@@ -10,6 +10,7 @@ namespace Airbnb.Domain.Entities
 {
 	public class Booking : BaseEntity<int>
 	{
+		//
```

Comment:
don't forget to add the ownership entity

Reply:
ok